test/asm/undefined-builtins.err

```diff
@@ -1,9 +1,9 @@
 error: undefined-builtins.asm(3):
-    PC has no value outside a section
+    PC has no value outside of a section
 error: undefined-builtins.asm(4):
     Interpolated symbol "@" does not exist
 error: undefined-builtins.asm(8):
-    _NARG does not make sense outside of a macro
+    _NARG has no value outside of a macro
```

Comment:
Does it exist at all? (What does `DEF()` report? What *should* it report?)

Reply:
`DEF(_NARG)` reports false, as it always has for years, and as this very test case demonstrates right below this line.

Reply:
I was going to suggest “does not exist” as an error message, but then I realised that things are a little messy:
```console
% rgbasm - <<EOF
println DEF(_NARG)
def _NARG equ 0
EOF
$0
error: <stdin>(2):
    '_NARG' already defined at <builtin>
error: Assembly aborted (1 error)!
```

"defined but not defined"? Hmm.